From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Mon, 2 Sep 2019 09:46:20 +0200
Subject: librem5: bookmarks-popover: Make the tag detail label adaptive

This helps the popover to fit in narrow screens.

Upstreamed in https://gitlab.gnome.org/GNOME/epiphany/-/merge_requests/833
---
 src/resources/gtk/bookmarks-popover.ui | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/resources/gtk/bookmarks-popover.ui b/src/resources/gtk/bookmarks-popover.ui
index 3e1498d..4f9adf6 100644
--- a/src/resources/gtk/bookmarks-popover.ui
+++ b/src/resources/gtk/bookmarks-popover.ui
@@ -111,8 +111,9 @@
                     </child>
                     <child>
                       <object class="GtkLabel" id="tag_detail_label">
-                        <property name="halign">center</property>
+                        <property name="ellipsize">end</property>
                         <property name="hexpand">true</property>
+                        <property name="max-width-chars">0</property>
                         <property name="visible">true</property>
                         <attributes>
                           <attribute name="weight" value="bold"/>
